model/monitorCache/implementation/DeliveryMonitoringData.php

```diff
@@ -200,12 +200,40 @@ private function getProgress()
         $session = $this->getTestSession();
 
         if ($session !== null) {
-            $pos = $session->getRoute()->getPosition();
-            $count = $session->getRouteCount();
-
             if ($session->isRunning()) {
-                $section = $session->getCurrentAssessmentSection();
-                $result =  __('%1$s - item %2$s/%3$s', $section->getTitle(), $pos+1, $count);
+                $route = $session->getRoute();
+                $routeItems = $route->getAllRouteItems();
```

Comment:
I think that instead of getting all items and loop through them is better to get item from current section using `Route::getRouteItemsByAssessmentSection()`:
https://github.com/oat-sa/qti-sdk/blob/legacy/qtism/runtime/tests/Route.php#L972